packages/ndla-ui/src/User/index.ts

```diff
@@ -7,5 +7,7 @@
  */
 
 import AuthModal from './AuthModal';
-
-export default AuthModal;
+import { UserInfo } from './UserInfo';
+import type { FeideGoGroup, FeideGroup, FeideOrg, FeideUserApiType } from './apiTypes';
+export { AuthModal, UserInfo };
```

Comment:
Dette fører til en breaking endring i ndla-ui. Du kunne fortsatt eksportert AuthModal som default og UserInfo som ekstra for å unngå det.

Reply:
Det skal være fikset nå 👍🏻